Resources/Audio/Backmen/Items/handling/attributions.yml

```
@@ -0,0 +1,71 @@
+- files: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)